frontend/src/views/DifferentialExpression/components/Main/components/DeResults/connect.ts

```diff
@@ -46,6 +51,15 @@ export const useConnect = ({ setIsLoading }: Props) => {
     setDifferentialExpressionResults(formattedResults);
   }, [rawDifferentialExpressionResults, isLoadingRaw, organismId]);
 
+  // Set error message if there is an error and the data is not loading
```

Comment:
given that error message is a primitive value (either a string or null), I think we can just assign a variable here without using `useEffect` to save us a render cycle

e.g.,

```ts
const errorMessage = isLoadingRaw ? null : errorMessage || null;
```